R/plot_time_series.R

```diff
@@ -1,7 +1,11 @@
 #' Plot time-series based on the results of Farrington Flexible
```

Comment:
maybe we should also update this. Results of a signal detection algorithm, being alarms, threshold and exptectation